index.html

```diff
@@ -0,0 +1,223 @@
+              class="location-icon"
+              src="./assets/icons/location.svg"
+              alt="location"
+            />
+            <p class="location-description poppins">
+              Silicon Valley, CA, United States of America
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="program-section" id="program">
+      <h2 class="program-text lato">Main Program</h2>
+      <div class="orange-divisor"></div>
+      <div class="cards-container">
+        <div class="card">
+          <img
+            class="card-icon"
+            src="./assets/icons/capstone-icons/original01.png"
+            alt=""
+          />
+          <h3 class="card-title lato">Lecture</h3>
+          <p class="card-description lato">
+            Listen to the speakers from various countries about AI in todays
+            world.
+          </p>
+        </div>
+        <div class="card">
+          <img
+            class="card-icon"
+            src="./assets/icons/capstone-icons/original02.png"
+            alt=""
+          />
+          <h3 class="card-title lato">Exhibition</h3>
+          <p class="card-description lato">
+            Appreciate various creations and projects applying AI concepts from
+            lectures.
+          </p>
+        </div>
+        <div class="card">
+          <img
+            class="card-icon"
+            src="./assets/icons/capstone-icons/original03.png"
+            alt=""
+          />
+          <h3 class="card-title lato">Forum</h3>
+          <p class="card-description lato">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+        <div class="card">
+          <img
+            class="card-icon"
+            src="./assets/icons/capstone-icons/original04.png"
+            alt=""
+          />
+          <h3 class="card-title lato">Workshop</h3>
+          <p class="card-description lato">
+            Utilize our latest creation to jumpstart your own project from
+            scratch.
+          </p>
+        </div>
+        <div class="card">
+          <img
+            class="card-icon"
+            src="./assets/icons/capstone-icons/original05.png"
+            alt=""
+          />
+          <h3 class="card-title lato">Ignite</h3>
+          <p class="card-description lato">
+            Global affiliate networking pre- and post-conference for
+            professional development.
+          </p>
+        </div>
+      </div>
+      <button class="join-btn lato">Join ArtificialEdge Conference</button>
+    </section>
+    <section class="speakers-section">
+      <h2 class="speakers-title lato">Featured Speakers</h2>
+      <div class="orange-divisor"></div>
+      <div class="speakers-container" id="speakers-container"></div>
+      <button class="see-more-btn lato" id="see-more">
+        MORE <img src="./assets/icons/arrow.svg" alt="" />
+      </button>
```

Comment:
- [ ] The _**MORE**_ button should only be seen on the mobile version of the Home page based on the design template. The button is intuitive for only mobile versions because of the space and convenience of users scrolling on mobile. Whereas, on the desktop version, it should display all the speakers at a glance based on the design on Figma. Kindly implement this change. 👍 

### Your Design (Desktop Version)

![image](https://user-images.githubusercontent.com/77106962/233155108-02d32bdd-1f67-4ea6-add3-1bff0870e7ce.png)

### Behance Design (Desktop Version)

![image](https://user-images.githubusercontent.com/77106962/233155379-de5c8062-4f8d-4490-97f6-33aeb1811429.png)